src/api/helpers/UC4Identifier.ts

```diff
@@ -7,7 +7,8 @@ export enum UC4Identifier {
 
     /* TRANSACTIONS */
     //common
-    TRANSACTION_APPROVAL = "approveTransaction",
+    TRANSACTION_APPROVAL = "approveOperation",
+    TRANSACTION_REJECTION = "rejectOperation",
```

Comment:
May rename the enum entry to `OPERATION_APPROVAL` etc

Reply:
It is still a transaction to approve an operation. In this enum, all transactions are prefixed with transaction